code-challenges.js

```diff
@@ -7,8 +7,47 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
 
-
-
+//create a function that takes in an array
+const removeShuffle = (array) => {
+    //shows the recursive function
+    console.log(array);
+    let newArr = []
```

Comment:
It looks like this newArr variable is never used in the function. We want to watch out for leaving orphaned code just lingering about!